visgraph/graphcore.py

```diff
@@ -310,9 +310,9 @@ def delNode(self, node):
         Example:
             g.delNode(node)
         '''
-        for edge in self.getRefsFrom(node):
+        for edge in self.getRefsFrom(node)[:]:
```

Comment:
Heh.  Just perf tested copying a list via:

y = list(x)
vs
y = x[:]

and am surprised to find that x[:] is faster :)